charts/asset-manager/values.yaml

```diff
@@ -51,31 +51,31 @@ appResources:
     memory: 1Gi
   requests:
     cpu: 500m
-    memory: 1Gi
```

Comment:
In the last commit message, please elaborate on _how_ you did the 'recent testing' and link to any graphs/resources that show the last 7 days' usage?